gateway-ha/src/main/java/io/trino/gateway/ha/resource/EntityEditorResource.java

```diff
@@ -86,8 +95,13 @@ public Response updateEntity(
                     ProxyBackendConfiguration backend =
                             OBJECT_MAPPER.readValue(jsonPayload, ProxyBackendConfiguration.class);
                     gatewayBackendManager.updateBackend(backend);
-                    log.info("Setting up the backend %s with healthy state", backend.getName());
-                    routingManager.updateBackEndHealth(backend.getName(), backend.isActive());
+                    log.info("Turning cluster %s %s", backend.getName(), backend.isActive() ? "on" : "off");
```

Comment:
on/off may be a little confusing, let's stick to the same terminology used in the code? 
```java
                    TrinoHealthStateType healthState = backend.isActive() ? TrinoHealthStateType.PENDING : TrinoHealthStateType.UNHEALTHY;
                    log.info("Marking cluster '%s' with health state %s", backend.getName(), healthState);
                    routingManager.updateBackEndHealth(backend.getName(), healthState);
...
```

Reply:
I think on/off is fine here because this code is triggered when user turns on/off cluster in the web UI. If we use the `healthState`, it's going to say "Marking cluster cluster_a with health state PENDING", which is not user-friendly IMO

Reply:
I agree with @xkrogen .. the error message is is misleading .. the cluster is not shut down or anything .. its just flagged with a specific status from the point of view of the Trino Gateway

Reply:
what about "Marking the cluster activated/deactivated"?